internal/services/storage/client/client.go

```diff
@@ -295,13 +378,16 @@ func (client Client) TablesClient(ctx context.Context, account accountDetails) (
 		return nil, fmt.Errorf("retrieving Account Key: %s", err)
 	}
 
-	storageAuth, err := autorest.NewSharedKeyAuthorizer(account.name, *accountKey, autorest.SharedKeyLiteForTable)
+	storageAuth, err := auth.NewSharedKeyAuthorizer(account.name, *accountKey, auth.SharedKeyTable)
```

Comment:
We can't do this in this API version, but can we add a TODO for this?

```suggestion
	// TODO: in API Version 2023-11-03 we can enable AAD Authentication here
	storageAuth, err := auth.NewSharedKeyAuthorizer(account.name, *accountKey, auth.SharedKeyTable)
```